Makefile

```diff
@@ -276,6 +274,12 @@ validate-mockgen: gen-code
 	git diff --quiet -- pkg/kv/mock/store.go || (echo "Modification verification failed! pkg/kv/mock/store.go"; false)
 	git diff --quiet -- pkg/pyramid/mock/pyramid.go || (echo "Modification verification failed! pkg/pyramid/mock/pyramid.go"; false)
 
+.PHONY: validate-apigen
+validate-apigen: gen-api
```

Comment:
Might want to add this also to checks-validator target

Reply:
Done